app/src/pages/inside/common/infoPanel/logViewSwitcher/index.js

```diff
@@ -0,0 +1,2 @@
+export { LogViewSwitcher } from './logViewSwitcher';
+export { LIST_VIEW, LOG_VIEW } from 'controllers/testItem/constants';
```

Comment:
It's better to import these constants from controllers/testItem where needed instead of re-exporting here